hydroshare/local_settings.py

```diff
@@ -198,9 +197,3 @@
 RECAPTCHA_SITE_KEY="6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
 RECAPTCHA_SECRET_KEY="6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
 RECAPTCHA_VERIFY_URL='https://www.google.com/recaptcha/api/siteverify'
-
-with open(os.path.dirname(os.path.abspath(__file__))  + "/../config/hydroshare-config.yaml", 'r') as stream:
```

Comment:
@hyi - did you intend to remove this.  This was added recently by @aphelionz to expose some config for @phuongdm and @alvacouch.

Reply:
@sblack-usu No, I did not intend to remove that at all. Not even sure how it got in. Could you take care of adding them back in? Thanks.

Reply:
@sblack-usu could you add these back in your PR https://github.com/hydroshare/hydroshare/pull/2963 since it modified local_settings.py?

Reply:
done